plugins/scaffolder-backend/package.json

```diff
@@ -71,6 +71,7 @@
     "octokit": "^2.0.0",
     "octokit-plugin-create-pull-request": "^3.10.0",
     "p-limit": "^3.1.0",
+    "prom-client": "^14.1.0",
```

Comment:
Sorry to be a pest, but could we make this `^14.0.1`? That's what the rest of the repo uses, and we don't want to get unintended duplicates.

Reply:
not a pest at all, thanks for the catch